src/stars.css

```diff
@@ -0,0 +1,36 @@
+/* #region stars */
+.stars {
+  display: flex;
+  height: 16px;
+}
+
+.stars__star {
+  height: 16px;
+  width: 16px;
+  background-image: url(./images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  border-radius: 0.5px;
```

Comment:
Do you really need this property? 
You are probably doing something wrong if you need to use float values for pixels